core/tests/test_utils.py

```diff
@@ -273,15 +270,15 @@ def get_pending_tasks(self, queue_name=None):
             list(Task). List of tasks in a single queue or in the entire
             taskqueue.
         """
-        return self.client.get_tasks(queue_name=queue_name)
+        return self._client.get_tasks(queue_name=queue_name)
 
 
 class MemoryCacheServicesStub(python_utils.OBJECT):
     """The stub class that mocks the API functionality offered by the platform
     layer, namely the platform.cache cache services API.
     """
 
-    cache_dict = collections.defaultdict()
+    _CACHE_DICT = {}
```

Comment:
Convention in the codebase is UPPER_CAMEL_CASE for class-level constants. I've also added an underscore prefix because it's private.